packages/web-components/src/global/color-mode.css

```diff
@@ -668,6 +668,28 @@ ic-theme,
   --ic-skeleton-background: var(--ic-architectural-100);
   --ic-skeleton-background-secondary: var(--ic-architectural-200);
 
+  /* Side navigation  */
```

Comment:
If these have been moved to colors.css, do they still need to be in this file? None of the others like this (Toast, Hero, etc.) are in here.